### DISCUSSION TITLE
Attribute Picker - webGIS 6

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#176] [General]
Wir haben die Editierdienste in einem eigenen CMS (wgpe) parametrisiert. 

Nun wollte ich mit dem Attribute-Picker Adress-Informationen aus Diensten, die im "Standard"-CMS (default ohne Bezeichnung) parametrisiert sind, auf ein Objekt "übertragen", leider bekomme ich folgende Fehlermeldung:

![image](https://github.com/gis-eni/webgis-forum/assets/1842500/ea9266e8-423b-40d1-b5d8-1a1e6e3bc69f)

Kannst du den Attribute Picker so gestalten, dass man auch cms-übergreifend Attribute "übertragen" kann?
... mit den Sonderfall, dass unser default-CMS keine Bezeichnung besitzt.

**Comment by gis-eni** [comment]
Der Dienst, auf den sich der Attribut Picker bezieht muss im gleich CMS sind. 
Eine Änderung wäre hier leider ein *Breaking Change*... Parametrierte Picker könnten bei einigen dann nicht mehr funktionieren.

Workaround: Dienst noch einmals neue CMS einbinden.

**Reply by gis-eni** [reply]
bzw das eine Thema noch in einen Dienst im gleichen CMS irgendwo einbinden

**Reply by marko-jer** [reply]
Schade, es würde einiges an doppelter Parametrierung ersparen. Es wäre abzuwägen, was in weiterer folge mehr Aufwand ist, AttributePicker einmal upzudaten oder Dienste oder Themen zusätzlich zu parametrisieren.

PS: Workaround ist mir klar ;)

**Reply by gis-eni** [reply]
Ok, dann sagen wir, es kommt, aber nicht in absehbarer Zeit. Parametrier halt einmal alles im gleichen CMS.
